src/Tizen.NUI.Components/Controls/Control.cs

```diff
@@ -42,6 +44,8 @@ public class Control : VisualView
 
         private bool onThemeChangedEventOverrideChecker;
 
```

Comment:
Thank you for your work. 
But could you resolve the warning shown below? 
I know this is not your modification, but basically we are going without build warning.
=>
src/Tizen.NUI.Components/Controls/Control.cs#L38
Use nameof in place of string literal 'Command'

Reply:
> Thank you for your work.
> But could you resolve the warning shown below?
> I know this is not your modification, but basically we are going without build warning.
> =>
> src/Tizen.NUI.Components/Controls/Control.cs#L38
> Use nameof in place of string literal 'Command'

Done